packages/zowe-explorer/i18n/sample/src/Profiles.i18n.json

```diff
@@ -1,6 +1,6 @@
 {
   "checkCurrentProfile.error": "Error encountered in ",
-  "addSession.quickPickOption": "Choose \"Create new...\" to define a new profile or select an existing profile to add to the DATA SETS Explorer",
+  "addSession.quickPickOption": "Choose \"Create new...\" to define or select a profile to add to the Job Views Explorer",
```

Comment:
I have a question about this update that came in from `master` branch and showed up running `yarn workspace vscode-extension-for-zowe vscode:prepublish`